.travis.yml

```diff
@@ -8,9 +8,22 @@ before_install:
 - yarn global add codecov
 
 script:
+- node -e "assert.ok(!/^(?:[67]|8\.[01])\./.test(require('@angular/compiler').VERSION.full))"
```

Comment:
Since we pinned its version in `package.json` already, this assertion seems unnecessary.

Reply:
Done.